mod/badges/set/abstract/award_badges.rb

```diff
@@ -20,10 +20,14 @@ def earns_badge action
 
 def award_badge badge_card
   name_parts = [Auth.current, badge_card.badge_type, :badges_earned]
-  badge_pointer =
-    subcard(name_parts) ||
-      attach_subcard(Card.fetch(name_parts, new: { type_id: PointerID }))
+  badge_pointer = Card.fetch(name_parts, new: { type_id: PointerID })
+  if ActManager.include? badge_pointer.name
+    director = ActManager.fetch(badge_pointer)
+    director.reset_stage
+    badge_pointer = director.card
+  end
```

Comment:
So do you think we can take out the ActManager part?

If it stays in, I think we should be able to explain the cases in which a mod developer would be expected to use this kind of thing, no?